### PR TITLE
IE fixes for IE8 and 7

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/javascripts/application.js.coffee [hunk 1]
```diff
@@ -317,32 +317,35 @@ jQuery ->
...
-          element[eventName] = undefined
-        element.removeAttribute(eventName)
-
-    element = null
-    return isSupported
+    if $("html").hasClass "lte-ie8"
+      return false
+    else
+      TAGNAMES = {
+                  "select": "input",
```
Comment (matthewford) [inline app/assets/javascripts/application.js.coffee]: Line contains inconsistent indentation

File: app/assets/stylesheets/frontend/forms.scss [hunk 2]
```diff
@@ -108,7 +108,9 @@ input[type="file"] {
   right: 0;
   margin: 0;
   cursor: pointer;
-  font-size: 200px;
+  font-size: 200px !important;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: !important should not be used

File: app/assets/stylesheets/frontend/forms.scss [hunk 3]
```diff
@@ -108,7 +108,9 @@ input[type="file"] {
...
+  -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)" !important;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: !important should not be used<br>Properties should be ordered position, top, right, margin, cursor, font-size, opacity, -ms-filter, filter, direction<br>Avoid vendor prefixes.

File: app/assets/stylesheets/frontend/forms.scss [hunk 4]
```diff
@@ -108,7 +108,9 @@ input[type="file"] {
...
+  filter: alpha(opacity=0) !important;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: !important should not be used

File: app/assets/stylesheets/frontend/layout.scss [hunk 5]
```diff
@@ -73,6 +73,11 @@
 
   &.page-header-wider div {
     width: 100%;
+
+    .lte-ie7 & {
+      width: auto !important;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/layout.scss]: !important should not be used

File: app/assets/stylesheets/frontend/layout.scss [hunk 6]
```diff
@@ -73,6 +73,11 @@
...
+      margin-right: 0 !important;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/layout.scss]: !important should not be used

**Comment by jaleszek** [comment]
1. I don't see the correlation with failed specs - good to merge
2. going to check specs after that

**Pull request merged**